packages/notebook/test/actions.spec.ts

```diff
@@ -1111,6 +1112,26 @@ describe('@jupyterlab/notebook', () => {
         expect(result).toBe(true);
         expect(widget.isSelected(widget.widgets[2])).toBe(true);
       });
+
+      it('should run the selected cells', async () => {
+        let emitted = 0;
+        let notebook: Notebook | null = null;
+        let lastCell: Cell | null = null;
+        NotebookActions.selectionExecuted.connect(async (_, args) => {
+          notebook = args.notebook;
+          lastCell = args.lastCell;
+          emitted += 1;
+        });
+        const result = await NotebookActions.runCells(
+          widget,
+          [widget.widgets[1], widget.widgets[2]],
+          sessionContext
+        );
+        expect(result).toBe(true);
+        expect(emitted).toBe(1);
+        expect(notebook).not.toBe(null);
+        expect(lastCell).not.toBe(null);
```

Comment:
I am always uneasy with `not.toBe(null)` because `undefined` technically is not a `null`. See https://github.com/adobe/brackets/issues/5479

```suggestion
        expect(notebook).toBeInstanceOf(Notebook);
        expect(lastCell).toBeInstanceOf(Cell);
```